### ISSUE TITLE
Docs mistake: enabling feature gates

### ISSUE THREAD

**Issue by artem-nefedov** [open-telemetry/opentelemetry-operator#2390]
### What happened?

Documentation currently contains an example on how to enable a feature gate via values in helm chart:

```sh
helm install opentelemetry-operator open-telemetry/opentelemetry-operator --set manager.featureGates=operator.autoinstrumentation.multi-instrumentation=enabled
```

If you try to use this example, controller won't be able to start, and will throw an error:

```
invalid argument "operator.autoinstrumentation.multi-instrumentation=enabled" for "--feature-gates" flag: no such feature gate "operator.autoinstrumentation.multi-instrumentation=enabled"
```

According to description of `--feature-gates` option, the correct way is without "=enabled" at the end (confirmed to work): 

```
helm install opentelemetry-operator open-telemetry/opentelemetry-operator --set manager.featureGates=operator.autoinstrumentation.multi-instrumentation
```

>      --feature-gates flag                               Comma-delimited list of feature gate identifiers. Prefix with '-' to disable the feature. '+' or no prefix will enable the feature. (default operator.autoinstrumentation.apache-httpd,operator.autoinstrumentation.dotnet,-operator.autoinstrumentation.go,operator.autoinstrumentation.java,-operator.autoinstrumentation.multi-instrumentation,-operator.autoinstrumentation.nginx,operator.autoinstrumentation.nodejs,operator.autoinstrumentation.python,operator.collector.rewritetargetallocator,-operator.observability.prometheus)

### Kubernetes Version

-

### Operator version

v0.89.0

### Collector version

-

**Comment by jaronoff97** [comment]
Yeah good catch – do you want to make the fix given it's minor? If not, I'll see if anyone else can make the change.

**Comment by artem-nefedov** [comment]
Sure, I'll post a PR
